src/NuGet.Core/NuGet.Protocol.Core.v3/HttpSource/HttpSource.cs

```diff
@@ -70,34 +71,43 @@ public HttpSource(PackageSource source, Func<Task<HttpHandlerResource>> messageH
             _messageHandlerFactory = messageHandlerFactory;
             _retryHandler = new HttpRetryHandler();
         }
-
-        internal Task<HttpSourceResult> GetAsync(string uri,
-            string cacheKey,
-            HttpSourceCacheContext context,
-            ILogger log,
-            CancellationToken cancellationToken)
-        {
-            return GetAsync(uri, cacheKey, context, log, ignoreNotFounds: false, cancellationToken: cancellationToken);
-        }
-
+        
         /// <summary>
         /// Caching Get request.
         /// </summary>
-        internal async Task<HttpSourceResult> GetAsync(string uri,
+        public async Task<HttpSourceResult> GetAsync(string uri,
```

Comment:
This is a rather large change and I would like to address this later, when we are cleaning up all of the resource retry loops.